docs/mobile-apps/features/device-vitals.md

```diff
@@ -0,0 +1,43 @@
+---
+id: device-vitals
+title: Device Vitals
+---
```

Comment:
```suggestion
sidebar_label: Device Vitals
---
```

Reply:
As per our style guide, we should also include a 'sidebar_label' at the top of the docs page: https://docs.saucelabs.com/contributing/style-guide/#frontmatter